cfg/cfg.go

```diff
@@ -62,11 +62,6 @@ var (
 	// RegistryAddress address of the plugin registry, if plugin IDs are specified in config instead of metas.
 	// Used as a default registry if a plugin fully-qualified name does not specify a registry.
 	RegistryAddress string
-
-	// DownloadMetas specifies whether the broker should download plugin metas from the registry. If
-	// true, then config file should be a list of plugin fully-qualified names. Otherwise, the config
-	// file should contain the plugin metas to be processed.
-	DownloadMetas bool
```

Comment:
It was needed for backward compatibility and not it is not needed anymore, right?

Reply:
Yes